test/parallel/test-module-version.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// Check for existence
+assert(process.config.variables.hasOwnProperty('node_module_version'));
+
+// Ensure that `node_module_version` is an Integer
+assert(!Number.isNaN(parseInt(process.config.variables.node_module_version)));
+assert.strictEqual(process.config.variables.node_module_version, 46);
```

Comment:
Probably better to test for number or `version >= 46` so it's not a maintenance burden.